draft-ietf-quic-manageability.md

```diff
@@ -587,6 +589,48 @@ Work is currently underway in the TLS working group to encrypt the SNI in TLS
 would make SNI-based application identification impossible through passive
 measurement.
 
+### Extracting Server Name Indication (SNI) Information
+
+If the SNI is not encrypted it can be derived form the QUIC Initial packet 
+by calculating the Initial Secret to decrypt the packet payload and parse the 
+QUIC CRYPTO Frame with the TLS ClientHello.
```

Comment:
```suggestion
QUIC CRYPTO Frame containing the TLS ClientHello.
```

Reply:
("with" sounds a little like one uses a ClientHello to parse the crypto frame, which would be a neat trick. :) )